docs/content/users/extend/additional-services.md

```diff
@@ -79,29 +79,29 @@ For example,
 
 Officially-supported add-ons:
 
-* [Adminer](https://github.com/ddev/ddev-adminer): `ddev get ddev/ddev-adminer`.
-* [Apache Solr for Drupal](https://github.com/ddev/ddev-drupal-solr): `ddev get ddev/ddev-drupal-solr`.
-* [Beanstalkd](https://github.com/ddev/ddev-beanstalkd): `ddev get ddev/ddev-beanstalkd`.
-* [Browsersync](https://github.com/ddev/ddev-browsersync): `ddev get ddev/ddev-browsersync`.
-* [cron](https://github.com/ddev/ddev-cron): `ddev get ddev/ddev-cron`.
-* [Elasticsearch](https://github.com/ddev/ddev-elasticsearch): `ddev get ddev/ddev-elasticsearch`.
-* [Memcached](https://github.com/ddev/ddev-memcached): `ddev get ddev/ddev-memcached`.
-* [MongoDB](https://github.com/ddev/ddev-mongo): `ddev get ddev/ddev-mongo`.
-* [PDFreactor](https://github.com/ddev/ddev-pdfreactor): `ddev get ddev/ddev-pdfreactor`
-* [Platform.sh](https://github.com/ddev/ddev-platformsh): `ddev get ddev/ddev-platformsh`
-* [Proxy Support](https://github.com/ddev/ddev-proxy-support): `ddev get ddev/ddev-proxy-support`.
-* [Redis Commander](https://github.com/ddev/ddev-redis-commander): `ddev get ddev/ddev-redis-commander`.
-* [Redis](https://github.com/ddev/ddev-redis): `ddev get ddev/ddev-redis`.
-* [Selenium Standalone Chrome](https://github.com/ddev/ddev-selenium-standalone-chrome): `ddev get ddev/ddev-selenium-standalone-chrome`.
-* [Varnish](https://github.com/ddev/ddev-varnish): `ddev get ddev/ddev-varnish`.
+* [Adminer](https://github.com/ddev/ddev-adminer): `ddev add-on get ddev/ddev-adminer`.
+* [Apache Solr for Drupal](https://github.com/ddev/ddev-drupal-solr): `ddev add-on get ddev/ddev-drupal-solr`.
+* [Beanstalkd](https://github.com/ddev/ddev-beanstalkd): `ddev add-on get ddev/ddev-beanstalkd`.
```

Comment:
Beanstalkd is gone
```suggestion
```

Please do a `ddev add-on list` to get the correct contents of this section.

Reply:
I'd rather not touch things that aren't related to what this PR is for. If we remove that then we need to remove from the table as well, and maybe the spellcheck list.... I'd much rather that all be done in a separate piece of work, probably along side work to automate it.

Reply:
The fix is just to do `ddev add-on list` and inject it here, since the format changed. You don't have to care about beanstalkd.

Reply:
It's probably just that copy-pasting this list is silly and we need to move to the add-on registry.